DietPi-Set_Software | Fix locales?

MichaIng opened the pull request:
Fixes https://github.com/Fourdee/DietPi/issues/1385#issuecomment-358089280

But I still would consider the following:

- Keep all additional user selected languages, just add `en_GB.UTF-8`, if it was missing.
Move `LANG=en_GB.UTF-8` out of `dietpi-globals` (into all dedicated scripts again), as we source it on every login which might affect also desktop/software languages? We can assure on the other hand that `dietpi-globals` itself is parsed correctly, if we set `LANG=en_GB.UTF-8` in the dedicated scripts **before** sourcing it. Just at boot/dietpi-login script, we might need to test, if everything behaves as expected. I am not sure how all this is handled before user login.
**€:** Ah, we also would need to set LANG in all global functions as local variable I guess 🤔.

Commit:
v6.0

DietPi-Set_Software | Fix locales?

Fourdee commented:
@MichaIng 

Many thanks 👍 

We need to match PREP with these changes aswell?
https://github.com/Fourdee/DietPi/blob/testing/PREP_SYSTEM_FOR_DIETPI.sh#L68-L99

Fourdee commented:
And we also need to consider removal (and maybe regen of locals during v6.0 patch)?
```
rm /etc/environment
rm /etc/default/locale
dpkg-reconfigure -f noninteractive locales
```

MichaIng commented:
@Fourdee 
Let's verify first, that these changes work as expected until release, locales turned out to be hard to predict 😄. But yeah then we should remove the files to bring everyone on same stage. Actually does no image come with `/etc/environment` including something else than LANG?

€: DietPi-Prep adjusted accordingly + some minor adjustments. The apt clean + update was doubled (before we change something on sources), or was this intended?

Commit:
v6.0

+ Remove /etc/default/locales and /etc/environment adjustment on locale setup.
+ Some minor typos/wording/cleanup.
+ Doubled apt clean and update before repository adjustments.

Commit:
v6.0

+ Remove unnecessary double quotes, which might also cause issues.

Fourdee commented:
@MichaIng 

> The apt clean + update was doubled (before we change something on sources), or was this intended?

Good spot 👍 not required.

> Actually does no image come with /etc/environment including something else than LANG?

non-DietPi images? Unsure, will need to check.

Fourdee reviewed (approved):
👍

Fourdee commented:
@MichaIng 

Thanks 👍 

Outstanding:
- Check/cleanup/patch for v6.0
```
/etc/environment
/etc/default/locale
#dpkg-reconfigure -f noninteractive locales #possibly?
```
